lambdas/call-ticf-cri/src/main/java/uk/gov/di/ipv/core/callticfcri/CallTicfCriHandler.java

```diff
@@ -185,7 +185,7 @@ private Map<String, Object> callTicfCri(IpvSessionItem ipvSessionItem, ProcessRe
             LOGGER.info(
                     LogHelper.buildLogMessage(
                             "CI score is breaching threshold - setting VOT to P0"));
-            ipvSessionItem.setVot(VOT_P0);
+            ipvSessionItem.setVot(Vot.P0.name());
```

Comment:
I think it would be conveniant - and presumably values already stored will be correctly unmarshalled back to an enum at the point that change was introduced.
I think the majority of the code affected will be in tests - I think we only set the Vot in a small number of places, but that still might be valuable.

Reply:
Yep, that might make it easier to tell if it's a good idea as well